docs/web.md

```diff
@@ -37,12 +37,14 @@ See the [fastify documentation](https://www.fastify.io/docs/latest/Logging/) for
 ## Pino with Express
 
 ```sh
-npm install express-pino-logger
+npm install pino-http
 ```
 
 ```js
-const app = require('express')()
-const pino = require('express-pino-logger')()
+import express from 'express'
```

Comment:
Why has this example been changed to ESM syntax? This isn’t copy and pasteable without further instruction.

Reply:
Actually true! Good spot